### ISSUE TITLE
Service stuck without crash

### ISSUE THREAD

**Issue by YentlFrickx** [newrelic/newrelic-fluent-bit-output#120]
We are running the nri-bundle on our openshift clusters and notice that after a while the new-relic logging pods are 'stuck' and don't forward the logs to new-relic anymore. When we delete the pod, the new pod works as expected and continues tailing the log files. I tried debugging the pod but can't open a shell in it, I could however execute the fluent-bit command through: `/fluent-bit/bin/fluent-bit --version`, which correctly returned the version. My assumption is that somewhere the fluent-bit command is stuck without it crashing and making the container restart.

**Comment by jsubirat** [comment]
Hi @YentlFrickx ,

Thank you for reporting this. Could you please try to capture the `newrelic-logging` pod logs of one of these "blocked pods" with `kubectl logs <name-of-stuck-newrelic-logging-pod>?

On the other hand, please note that you can use the _debug_ images to diagnose these kind of issues. The _debug_ images come with the _busybox_ binary, which you can use to execute commands on that box, including `sh`. To use this image, you'll need this configuration in `values.yml`:
```
newrelic-logging:
  image:
    tag: 1.14.0-debug
```

Version 1.14.0 is the one currently being used by our Helm chart. Nevertheless, there are also 1.14.1 and 1.14.2 available that you can use if you overwrite the `newrelic-logging.image.tag`. In particular, version 1.14.1 includes a fix for [a Fluent Bit bug](https://github.com/fluent/fluent-bit/issues/5753#issuecomment-1241174476) that might be related to what you experience.

So, in summary, could you please:

1. Send us the newrelic-logging logs.
2. Overwrite the image tag to 1.14.0-debug and use `busybox` to get shell access (`busybox sh`) and diagnose the issue. List of Busybox commands [here](https://en.wikipedia.org/wiki/BusyBox#Commands).
3. Overwrite the image tag to 1.14.1 and check if that fixes your issue.

Best regards,

Josep
